src/main/java/services/ConnectionFactory.java

```diff
@@ -10,9 +10,9 @@
  * @author klysov
  */
 public class ConnectionFactory {
-    private static final String URL = "jdbc:mysql://localhost:3306/ticketsystem";
-    private static final String USER = "testuser";
-    private static final String PASS = "testpassword";
+    private static final String URL = "jdbc:mysql://localhost:3306/plane";
+    private static final String USER = "root";
+    private static final String PASS = "root";
```

Comment:
you have to have same database properties such as URL, USER, PASSWORD
it should not be changed in every commit

Reply:
I don't know any idea how solve this issue

Reply:
all of you just configure your dbs in the same way

Reply:
Done